src/modals/SendModal/TransactionForm.tsx

```diff
@@ -43,13 +43,17 @@ const SendModalTransactionForm = ({ data, onSubmit, onCancel }: TransactionFormP
   const { addresses, mainAddress } = useAddressesContext()
   const [fromAddress, setFromAddress] = useState(data?.fromAddress ?? mainAddress)
   const [toAddress, setToAddress] = useState(data?.toAddress ?? '')
+  const [addressError, setToAddressError] = useState('')
   const [amount, setAmount] = useState(data?.amount ?? '')
   const [gasAmount, setGasAmount] = useState(data?.gasAmount ?? '')
-  const [gasPrice, setGasPrice] = useState(data?.gasPrice ?? '')
-  const [addressError, setToAddressError] = useState('')
   const [gasAmountError, setGasAmountError] = useState<ReactNode>()
-  const minimalGasPriceInALPH = formatAmountForDisplay(MINIMAL_GAS_PRICE)
+  const minimalGasPriceInALPH = formatAmountForDisplay(MINIMAL_GAS_PRICE, true)
+  const [gasPrice, setGasPrice] = useState(data?.gasPrice ?? minimalGasPriceInALPH)
```

Comment:
Why not leaving it blank as it was before (`''`) to let the node specify the fee?

Reply:
Because if I remember correctly, when submitting a contract or script, it just wouldn't work properly with the values specified by the node.

Reply:
Have you reported this to the backend team? I'd be interested to see the conversation

Reply:
Yeah I did - the conversation is on Slack but it was pretty brief; to paraphase: "contract gas estimation is non-trivial" . 

But you bring up a good point of focus: this shouldn't always be set by default - only if dappTransactionData has contract data.